env-example

```diff
@@ -49,6 +49,7 @@ FEEDER_CDN_PRIVATE_HOST=p.staging.prxu.org
 ID_HOST=id.staging.prx.tech
 PLAY_HOST=play.staging.prx.tech
 PRX_HOST=beta.staging.prx.tech
+PUBLIC_FEEDS_PREFIX=https://publicfeeds.net/f
```

Comment:
Opted to make this a "prefix" rather than "host".  Since it also needs that `/f` path.  But I could be convinced otherwise.

Reply:
Okay with prefix (though in other places seems like we may use the term _base_ for this sort of thing), but may consider `PUBLIC_FEEDS_URL_PREFIX` to make it less ambiguous

Reply:
I was modeling on the feeder `feed.enclosure_prefix` column - which we fill in with this like `http://www.podtrac.com/pts/redirect.mp3`.

Yeah, will add `_URL` to that.